css/style.css

```diff
@@ -0,0 +1,155 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Coiny', cursive;
+    font-size: 1rem;
+    line-height: 1.5;
+    color: #AA6373;
+    overflow-x: hidden;
+    width: 100%;
+}
+
+nav {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    width: 100%;
+}
+
+ul {
+    float: right;
+    margin: 25px 20px 0 0;
+
+}
+
+li {
+    list-style: none;
+    display: inline;
+}
+
+a {
```

Comment:
I am guessing this is for contacts in later build. However it is currently not doing anything